lib/sycamore/sycamore/query/client.py

```diff
@@ -114,6 +117,15 @@ def __init__(
         self.os_client_args = os_client_args
         self.trace_dir = trace_dir
 
+        if context and os_client_args:
```

Comment:
Should we just get rid of these extra params and require them to be set in the context?

Reply:
I figured I'd do it separate since the UI is probably using it